### ISSUE TITLE
Warmer & Analytics

### ISSUE THREAD

**Issue by augpaschal** [nexcess/magento-turpentine#1006]
How google analytics and other web analytics tools see the traffic from warmer?

**Comment by miguelbalparda** [comment]
Can you rephrase your question?

**Comment by augpaschal** [comment]
the traffic generated by the warmer script is counted in statistics of google analytics?

And something else. Sitemap contains 5000 url and it needs some hours to parse all of these. any suggestions?

**Comment by miguelbalparda** [comment]
This is not related to Turpentine. You should read about GA and see how they handle bot visits but I suspect every hit count in GA. 

About the urls there is not much to suggest, I would say you should run the warmer late at night or when the site has not much traffic to avoid clogging the server.

**Comment by augpaschal** [comment]
What value is more suitable for "Default TTL" ? For above reason 1-2 hours i think isn't suitable.

**Comment by aricwatson** [comment]
Depending on how often your site is updated and how/when you flush items from the Varnish cache, setting a very long TTL may work well for you. This will vary quite a bit from site to site, so it's nothing something we can really make any general recommendations about.
